tests/unittests/GraphOptzTest.cpp

```diff
@@ -28,11 +28,34 @@ using namespace glow;
 
 class GraphOptz : public ::testing::Test {
 public:
-  GraphOptz() { F_ = mod_.createFunction("main"); }
+  GraphOptz() {
+    mod_ = &EE_.getModule();
+    originalF_ = mod_->createFunction("main");
+  }
 
 protected:
-  Module mod_;
-  Function *F_;
+  void checkNumericalEquivalence() {
```

Comment:
Question: 
What if the unoptimized function was producing some kind of result that is e.g. the same for all inputs (e.g. all zeroes), because it was never run for real before and nobody noticed. May be there is a node there that makes all outputs zeros... In this case, the optimized function would eventually produce also all zeros, but not necessarily because the functions are equivalent, but rather because this node that zeros everything is still there. Are we OK with this? Can we improve our checks/tests somehow to catch such situations?

Reply:
It feels hard to do this on a systematic basis since it really depends on the details of the Nodes.